src/profile_service/activity.details.js

```diff
@@ -80,6 +107,15 @@ module.exports.ActivityDetails = class ActivityDetails {
     return this.extendedProfile;
   }
 
+  /**
+   * The application profile object.
```

Comment:
Can we change this to `Profile of an application, with convenience methods to access well-known attributes.`?

Also as discussed, can we update the method description of the new profile above to reflect the fact that it is preferable, and contains anchor information too. Deprecating the old method too 👍

Reply:
Updated in 16c49a8